public/ipc.js

```diff
@@ -157,17 +155,39 @@ ipc.on('get job', async (event, data) => {
 })
 
 ipc.on('abort job', (event, data) => {
-  Object.values(queues[data.job.queue.name].childPool.retained).forEach((child) => {
-    child.send({__abortJobId__: data.job.id})
-  })
+  dialog_.showMessageBox(
+    {
+      type: 'question',
+      buttons: ['Cancel', 'Abort'],
+      message: `Are you sure you want to abort ${data.jobDescription}?`
```

Comment:
Just in case the job doesn't have a job description passed, let's change `${data.jobDescription}` to `${data.jobDescription || 'this task'}`.